Date: Tue, 27 Apr 2004 13:48:40 -0500 (CDT)
From: Olof Johansson <olof@austin.ibm.com>
To: akpm@osdl.org
Cc: anton@samba.org, <olh@suse.de>,
	<linuxppc64-dev@lists.linuxppc.org>
Subject: [PATCH] ppc64: Set memory-only nodes online

Hi,

On pSeries LPARs we might end up with NUMA nodes that only have memory and
no CPUs. Only the CPU configuration code actually set a node online, so
memory-only nodes wouldn't show up in sysfs. Below patch adds the
set_online call to the memory loop too.


-Olof


===== arch/ppc64/mm/numa.c 1.24 vs edited =====
--- 1.24/arch/ppc64/mm/numa.c	Mon Apr 12 12:54:46 2004
+++ edited/arch/ppc64/mm/numa.c	Mon Apr 26 22:51:55 2004
@@ -184,6 +184,8 @@
 		if (numa_domain >= MAX_NUMNODES)
 			BUG();

+		node_set_online(numa_domain);
+
 		if (max_domain < numa_domain)
 			max_domain = numa_domain;


